sdk/servicebus/azure-servicebus/tests/test_queues.py

```diff
@@ -1604,7 +1673,7 @@ def message_content():
                     yield message
 
             with sb_client.get_queue_sender(servicebus_queue.name) as sender:
-                message = ServiceBusMessageBatch()
+                message = ServiceBusMessageBatch(amqp_transport=amqp_transport)
```

Comment:
Is it the expectation that customers have to set the transport of the batch as well?
I thought we managed to avoid this for EventHubs?

Reply:
Ah nope, the expectation is that customers don't set this at all b/c they're creating batches through the "create_message_batch" on the client. This was just testing what would happen in the edge case of users manually passing in a SBMessageBatch (so PyamqpTransport by default) to a client that uses the UamqpTransport. I believe I wanted to test the opposite as well (batch uses uamqp, client uses pyamqp) just to see if it would work, even though a user would never be able to make this happen, and it does. I must've forgotten to remove the passing in of amqp_transport for the second case. Will update!